### PR TITLE
Clarify documentation. DAS-1450

### PR DESCRIPTION
**Pull request by flamingbear**
includes:
  * Renaming of functions and variables for clarity.
  * Updated README for docker or local execution.
  * Deleted a copy pasted version of the Harmony API Introduction notebook that was modified for a single person's computer.
  * Explicitly skip the tests in test_transform.py as they were passing incorrectly before.  This doesn't fix the tests, but just marks them as skipped.

  DAS-1450

### CODE REVIEW

File: tests/test_transform.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 """
 Description:
-This script does unit test for transform.py. It includes three tests:
+This script does units test for transform.py. It includes three tests:
```
Comment (owenlittlejohns) [inline tests/test_transform.py:3]: This is not a serious nit, but should this be "unit tests" rather than "units test"?

**Review by owenlittlejohns — CHANGES_REQUESTED** [review]
The README is definitely improving with each of these PRs, thank you!

The change in the function names in `transform.py` are really minor (but definitely clarifying), but where that's still a change to the service code itself, I think this warrants a new patch version in `versions.txt` (and the accompanying changes to `CHANGE.md`).

File: tests/test_transform.py (hunk 1 above)
Comment (flamingbear) [reply tests/test_transform.py:3]: Yes it should, which makes me think I missed something else that should have gone from unit to units.

Comment (flamingbear) [reply tests/test_transform.py]: 22bac53. No I just put the s in the wrong place.

Comment (owenlittlejohns) [reply tests/test_transform.py]: Cool. Thanks for the update.

**Review by owenlittlejohns — APPROVED** [review]
Thanks for updating the version and change-log.

**Pull request merged**